symbols/overlay29.yml

```diff
@@ -4561,15 +4610,15 @@ overlay29:
         
         r0: user entity pointer
         r1: target entity pointer
-    - name: HasConditionalGroundImmunity
+    - name: IsLevitating
```

Comment:
I don't really like this name. "Levitating" implies the levitate ability, which is notably not what this function does. That's why I called it `HasConditionalGroundImmunity`. It applies to things that can give ground immunity that aren't permanent.

I guess since this applies to more than just type matchups, we should still rename it though. Can we do something like `IsHovering` or `IsFloating`?

Reply:
Good point. I like `IsFloating`, I'll change it.